Feat/dv 943 reown starterkit (#281)
rsksmart/devportal: tools/tokenbridge/index.md

## Overview

> We have a bridge smart contract on each network, the bridge on one chain will receive and lock the ERC20 tokens, then it will emit an event that will be served to the bridge on the other chain. There is a Federation in charge of sending the event from one contract to the other. Once the bridge on the other chain receives the event from the Federation it mints the tokens on the mirror ERC20 contract.

Reviewer: Once the bridge on the other chain receives the event from the Federation, it mints the tokens on the mirror ERC20 contract

> We have a bridge smart contract on each network, the bridge on one chain will receive and lock the ERC20 tokens, then it will emit an event that will be served to the bridge on the other chain. There is a Federation in charge of sending the event from one contract to the other. Once the bridge on the other chain receives the event from the Federation, it mints the tokens on the mirror ERC20 contract.

Reviewer 2: ```suggestion
The smart contract on each network achieve interoperability using a bridge, the bridge on one chain will receive and lock the ERC20 tokens, then it will emit an event that will be served to the bridge on the other chain. A Federation is in charge of sending an event from one contract to the other. Once the bridge on the other chain receives the event from the Federation, it mints the tokens on the mirror ERC20 contract.
```

Reviewer 3: @owans This suggestion actually changes the meaning of what's being said. There are indeed two smart contracts (one deployed on each network), and your suggestion appears to imply that there is only one.

See the [FAQ](/tools/tokenbridge/faq/) to learn more about how it works!

<img src="/assets/img/tools/tokenbridge/token-bridge-diagram.png"/>

> The bridge contracts are upgradeable as we wish to move to a decentralized bridge in the future. Here is the first

Reviewer 2: ```suggestion
The bridge contracts are upgradeable, this enables a smoother move to a more decentralized bridge in the future. Here is the first
```

[POC of the trustless decentralized bridge](https://github.com/rsksmart/tokenbridge/releases/tag/decentralized-poc-v0.1)

## Usage

> You can use the ['Token Bridge Dapp'](https://tokenbridge.rsk.co/) together with [Nifty Wallet](https://chrome.google.com/webstore/detail/nifty-wallet/jbdaocneiiinmjbjlgalhcelgbejmnid) or [Metamask with custom network](https://github.com/rsksmart/rskj/wiki/Configure-Metamask-to-connect-with-RSK) to move tokens between networks. 

Reviewer: Same comment, wiki is down

> This is the [Dapp guide](/tools/tokenbridge/dappguide/) if you don't know how to use it.

Reviewer 2: ```suggestion
Follow the [Dapp guide](/tools/tokenbridge/dappguide/) for more details on how to use the token bridge.
```

Alternatively, you can use a wallet or web3js with the ABI of the contracts. See ['interaction guide using MyCrypto'](/tools/tokenbridge/usingmycrypto/) for more information on how to use the bridge.

## Developers

### Contracts

Here are the ['addresses'](/tools/tokenbridge/contractaddresses/) of the deployed contracts in the different networks.

> The smart contracts used by the bridge and the deploy instructions are in the token bridge repository on the ['bridge folder'](https://github.com/rsksmart/tokenbridge/tree/master/bridge/).

Reviewer: "in the folder", I guess

The ABI to interact with the contracts are in the ['abis folder'](https://github.com/rsksmart/tokenbridge/tree/master/abis)

### Federation

> There is a federation in charge of notifying the events that have happened in the bridge between one chain and the other. The federation is composed of the creators of the token contracts who want to enable their token for crossing.

Reviewer 2: ```suggestion
There is a federation in charge of notifying the events that have happened in the bridge between one chain and another chain. The federation is composed of the creators of the token contracts who want to enable their token for crossing.
```
